server/topic.go

```diff
@@ -512,21 +529,25 @@ func (t *Topic) runLocal(hub *Hub) {
 						}
 					}
 					// Update user's last online timestamp & user agent
-					if err := store.Users.UpdateLastSeen(asUid, mrs.userAgent, now); err != nil {
-						log.Println(err)
+					if !proxyTerminating {
+						if err := store.Users.UpdateLastSeen(asUid, mrs.userAgent, now); err != nil {
```

Comment:
Is it correct to use `asUid` vs `uid` here? I suppose they should be the same not 100% sure.

Reply:
Good catch. I think you are right. We are notifying the other users of an event on behalf of `uid`, not `asUid`.